src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -486,150 +512,151 @@ public static class PublishedContentExtensions
+            //                            }
+            //                            else
+            //                            {
+            //                                // Nothing is strong typed anymore.
+            //                                result = EnumerableInvocations.Cast(parameterType, (IEnumerable)converted);
+            //                            }
+            //                        }
+            //                        else
+            //                        {
+            //                            // Return single expected items from converters returning an IEnumerable.
+            //                            // Check for key/value pairs and strings.
+            //                            if (convertedType.IsEnumerableType()
+            //                                && !convertedType.IsEnumerableOfKeyValueType()
+            //                                && !(convertedType == typeof(string) && propertyType == typeof(string)))
+            //                            {
+            //                                // Use 'FirstOrDefault' to convert the type back to T.
+            //                                result = EnumerableInvocations.FirstOrDefault(
+            //                                    propertyType,
+            //                                    (IEnumerable)converted);
+            //                            }
+            //                            else
+            //                            {
+            //                                result = converted;
+            //                            }
+            //                        }
+            //                    }
+            //                    else
+            //                    {
+            //                        // Ensure we pass back an empty enumerable if the expected output is an enumerable.
+            //                        // and null has been returned by the type converter.
+            //                        if (propertyIsEnumerableType)
+            //                        {
+            //                            result = EnumerableInvocations.Empty(typeInfo.GenericTypeArguments.First());
+            //                        }
+            //                    }
+            //                }
+            //                else if (propertyType.IsInstanceOfType(propertyValue))
+            //                {
+            //                    // If the TypeConverter's `CanConvertFrom` has returned false,
+            //                    // then we can check if the value is the same type as the target type.
+            //                    result = propertyValue;
+            //                }
+            //            }
+            //        }
+            //    }
+            //}
+
+            //TODO: Move this to 
+
+            //if (propertyType == typeof(HtmlString))
```

Comment:
Having a bespoke check for `HtmlString` here has started to feel hacky.  I know `HtmlString` has common usage on websites, but I'm wondering if there's a better way to handle this scenario?